docs/user_guide.rst

```diff
@@ -0,0 +1,208 @@
+User Guide
+==========
+
+.. contents::
+
+Firefox
+*******
+
+To run your automated tests with Firefox, specify ``Firefox`` as your driver::
+
+  py.test --driver Firefox
+
+Chrome
+******
+
+To use Chrome as the driver, you need to have ChromeDriver installed. You can
+download it
+`here <https://sites.google.com/a/chromium.org/chromedriver/downloads>`_.
+You may either download ChromeDriver to a location in your PATH, or utilize
+the option to specify the driver path during test execution.
+
+To run your automated tests, specify ``Chrome`` as your driver::
+
+  py.test --driver Chrome
+
+If ChromeDriver is not on your path, use ``--driver-path`` to specify the
+location::
+
+  py.test --driver Chrome --driver-path /path/to/chromedriver
+
+For more information relating to ChromeDriver, you may read its documentation
+`here <https://sites.google.com/a/chromium.org/chromedriver/>`_.
+
+PhantomJS
+*********
+
+To use PhantomJS as the driver, you need to have it installed. You can download
+it `here <http://phantomjs.org/download.html>`_.
+You may either download PhantomJS to a location in your PATH, or utilize
+the option to specify the driver path during test execution.
+
+To run your automated tests, specify ``PhantomJS`` as your driver::
+
+  py.test --driver=PhantomJS
+
+If PhantomJS is not on your path, use ``--driver-path`` to specify the
+location::
+
+  py.test --driver=PhantomJS --driver-path=/path/to/phantomjs
+
+For more information relating to PhantomJS, you may read its documentation
+`here <http://phantomjs.org/quick-start.html>`_.
+
+Selenium Server/Grid
+********************
+
+To run your automated tests against a
+`Selenium server <https://github.com/SeleniumHQ/selenium/wiki/RemoteWebDriverServer>`_
+or a `Selenium Grid <https://github.com/SeleniumHQ/selenium/wiki/Grid2>`_ you
+must have a server running and know the host and port of the server.
+
+By default Selenium will listen on host 127.0.0.1 and port 4444. This is also
+the default when running tests against a remote driver.
+
+To run your automated tests, simply specify ``Remote`` as your driver. Browser
+selection is determined using capabilities. Check the
+`documentation <https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities#used-by-the-selenium-server-for-browser-selection>`_
+for details of accepted values. There are also a number of
+`browser specific capabilities <https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities#browser-specific-capabilities>`_
+that can be set. Be sure to also check the documentation for your chosen
+driver, as the accepted capabilities may differ.
+
+Note that if your server is not running locally or is running on an alternate
+port you will need to specify the ``--host`` and ``--port`` command line
+options.
+
```

Comment:
You provide a sample command line for running with `Firefox`, `Chrome` and `PhantomJS`, but none when using `Remote`. I realize that there are more options and combinations for this, but perhaps a sample command showing specifying `--driver`, `--host` and `--port` would be helpful?